microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,155 @@
+
+![Protocol Handler](./devtools-105-images/protocol-handler.png)
+
+See also:
+
+* [Test Progressive Web App (PWA) protocol handling](../../../progressive-web-apps/protocol-handlers.md)
+* [Handle protocols in Progressive Web Apps](../../../../progressive-web-apps-chromium/how-to/handle-protocols.md)
+
+
+<!-- ====================================================================== -->
+## Edit and resend network requests more reliably in the Network Console tool
+
+<!-- Title: Edit and resend network requests more reliably in the Network Console tool  -->
+<!-- Subtitle: Modify and resend network requests that have been logged in the Network tool with the Network Console tool. -->
+
+Previously in Microsoft Edge, selecting **Edit and resend** for a network request from the Network tool may not have reliably opened the **Network Console** tool and pre-populated it with the values from the request.
+
+In Microsoft Edge 105, this issue has been fixed.  **Edit and resend** now pre-populates the **Network Console** tool with the values for the network request that you are looking to resend. You can continue to modify these values before sending the request from the **Network Console** tool. Thank you for sending us your feedback about this issue!
+
+Right-click the network request that you want to change and resend, and then select **Edit and Resend**:
+
+![Edit and Resend](./devtools-105-images/edit-and-resend.png)
+
+In the **Network Console**, edit the network request information, and then click the **Send** button:
+
+![Network Console Edit and Send](./devtools-105-images/networkconsole-edit.png)
+
+See also:
+* [Compose and send web API requests using the Network Console tool](../../../network-console/network-console-tool.md#starting-from-the-network-tool)
+
+
+<!-- ====================================================================== -->
+## Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View
+
+<!-- Title: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View -->
+<!-- Subtitle: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. -->
+
+In Microsoft Edge 105, several improvements to location controls have been made in [Focus Mode](/devtools-guide-chromium/experimental-features/focus-mode), including a new option for changing the orientation of **Quick View**.
+
+The **Customize and control DevTools** (**...**) menu now directly provides options for changing the docking location of DevTools.  This menu also contains settings for where the **Activity Bar** is displayed.  The previous submenus have been replaced with toggle buttons.  Additionally, improved contrast means the option you have selected is more easily visible at a glance.
```

Comment:
This last sentence is phrased a little awkwardly. I tried to improve it but I'm fine with whatever you think is best.
```suggestion
The **Customize and control DevTools** (**...**) menu now directly provides options for changing the docking location of DevTools.  This menu also contains settings for where the **Activity Bar** is displayed.  The previous submenus have been replaced with toggle buttons.  Additionally, improved contrast means the currently selected option is more easily visible at a glance.
```